Spigot-Server-Patches/0611-Add-warning-for-servers-not-running-on-Java-11.patch

```diff
@@ -36,7 +41,11 @@ index 0000000000000000000000000000000000000000..023123d52fbb458e6ee2d38e42c9651a
 +
 +        final int javaVersionNum;
 +        try {
-+            javaVersionNum = Integer.parseInt(version);
++            Matcher versionMatcher = NUM_PATTERN.matcher(version);
++            if (!versionMatcher.find()) {
++                PaperJvmChecker.printWarning(javaVersion, logger);
```

Comment:
Perhaps return afterwards? Not sure if we should try to parse the number after it hasn't been found :p

Reply:
Yeah, for sure